src/components/DocumentLayouts.js

```diff
@@ -12,20 +12,36 @@ import {
 import MainLayout from './LayoutTypes/MainLayout'
 
 // import styles from './styles'
-import { BulletList as BulletListStyles, ColumnView as ColumnViewStyles, RowView as RowViewStyles } from './LayoutTypes/LayoutStyles'
-//import BulletListStyles from './LayoutStyles/BulletList'
-//import RowViewStyles from './LayoutStyles/RowViewStyles'
-//import ColumnViewStyles from './LayoutStyles/ColumnView'
+
+// import BulletListStyles from './LayoutStyles/BulletList'
+// import RowViewStyles from './LayoutStyles/RowViewStyles'
+// import ColumnViewStyles from './LayoutStyles/ColumnView'
+
+import { BulletList, ColumnView, RowView } from './LayoutTypes/LayoutStyles'
 
 // @TODO move away switch into a separated function
 
 const DocumentLayouts = (type, documentTitle, data) => {
     let layout
 
-    type === 'PDF1' ? layout = <MainLayout styles={BulletListStyles} data={data} /> :
-        type === 'PDF2' ? layout = <MainLayout styles={RowViewStyles} data={data} /> :
-            type === 'PDF3' ? layout = <MainLayout styles={ColumnViewStyles} data={data} /> :
-                layout = <MainLayout styles={BulletListStyles} data={data} />
+    switch (type) {
```

Comment:
ok, switch is back, but can you propose something better in this situation?
React have a lot of cool patterns, ways to improve it. take time and try to figure it out